docs/user-manual/onboarding-guide/programguide

```diff
@@ -0,0 +1,112 @@
+* You can simplify and optimize your program applications.  
+* You can simplify how you process and follow up on applications.
+
+To ensure a successful CiviForm implementation, read through the brief guide below and create your plan to build programs.
+
+## Understanding Your Needs and Goals
+
+- [ ] **Define user needs:** To create program forms that people will find intuitive and easy to use, consider the needs of both your residents and program administrators.  
+      - [ ] Spend time to understand how your residents interact with the program applications so that you can optimize the experience of CiviForm users (including residents and administrators):  
+            - [ ] Review [Preparing for Human-Centered Redesign](https://beeckcenter.georgetown.edu/report/preparing-for-human-centered-redesign/).  
+            - [ ] Read more about [mapping the applicant experience](https://github.com/usds/benefits-enrollment-prototype/blob/master/assets/discovery-findings-mapping-enrollment-Nov2016.pdf).  
+      - [ ] Consider engaging community-based organizations for user research.
+
+- [ ] **Decide who will be the [CiviForm Admin](https://docs.civiform.us/overview/how-does-civiform-work\#civiform-admin-experience)** across your civic entity  
+      - [ ] Decide who will own and maintain your CiviForm instance in the long term (IT team, Digital Services or other). This will be the person who creates the Forms and Programs on CiviForm.
+
+- [ ] **Review and evaluate** the [Recommended CiviForm Launch Team](https://docs.google.com/document/d/1AacsuDbMmVfmBuWWSNRz3nOUYR3WG5\_KDnDgxrc6z0A/edit)  
+      - [ ] Consider if any new staffing or resources are needed for the implementation / maintenance of CiviForm and/or the programs.
+
+- [ ] **Choose programs to put in CiviForm:** Determine which programs will benefit most from using CiviForm (e.g., benefit applications, complaint forms, event registrations, permit applications).  
+      - [ ] Evaluate programs most likely to benefit from CiviForm by using these resources:  
+            - [ ]  [CiviForm Fellowship - Program Assessment](https://docs.google.com/document/d/10TTk68evp-X8iJ2W3S2--MpiqMeg1Uf0jTysbZITEdg/edit?usp=sharing)  
+            - [ ]  [CiviForm - Program Evaluation Worksheet ](https://docs.google.com/spreadsheets/d/1\_NNMln-LOAxxXCoYgh1oI0uaS6YhoyMoBvudaiKRMm4/edit?resourcekey=0-KPl3Y1YgUYty81OKHbRMpQ\#gid=629433048)  
+      - [ ] Examine selected programs to identify opportunities to streamline overlapping questions:  
+            - [ ] Review the information gathered on the application for each program and remove any unnecessary questions.  
+            - [ ] Work with program stakeholders including program administrators to understand the administrative and policy requirements for different programs.   
+            - [ ] Identify common questions across programs (Ex. Name, Address, income).  
```

Comment:
**nitpick:** Keep capitalization the same on Name, Address, income

Reply:
Thanks. I will change this in the Google Doc.

Reply:
What is the plan to integrate the changes back into github? I imagine it will now be hard to keep track of what has changed in the google doc?

Reply:
The short term plan (following your advice) is to have the content complete before uploading any changes to github. If an error makes it on the site, use github workflow to fix.